integration-tests/features/environment.py

```diff
@@ -15,6 +15,10 @@
 # The following API endpoint is used to check if the system is started
 _API_ENDPOINT = 'api/v1/'
 
+# Ports used by various services
+_FABRIC8_ANALYTICS_SERVER = 32000
+_FABRIC8_ANALYTICS_JOBS = 34000
+_FABRIC8_MINIO_S3 = 33000
```

Comment:
Also probably remove "Minio" - we werer using different S3 local alternative but were forced to switch to Minio due to some features. Removing such names from variables can help in future when possible similar switch will be done.